### DISCUSSION TITLE
QHY QHYCFW3-L Filter wheel RJ-11 interface NOT Green

### DISCUSSION THREAD

**Discussion by JDAstroPhoto** [mworion/MountWizzard4#274] [Q&A] [answered]
I am using the QHY600 camera with the QHYCFW3-L filter wheel.  The filter wheel has no direct interface to the SimplyNUC  computer.  The power and control (RS232) is directly from the camera with a single cable.  The cable is RJ11 into the filter.
I use QHY ASCOM drivers.
On MW4 you provide QHY ASCOM drivers for the QHY filter wheel, but I can't get any of them to turn green.
I have a similar issue with the OPTEC FocusLynx focuser.  
I really don't need these to be operational in MW4, since they are set and forget for modeling, but I was just wondering if these should be working.
Thanks

**Comment by mworion** [comment, marked as answer]
If you are using ASCOM (I do use INDI), please be reminded that some ASCOM device drivers are not capable to be connected from multiple applications the same time. As you are using NINA my guess is that you connected them already through NINA. Unfortunately NINA exposes only the camera itself, so I cannot use NINA for interfacing FW and Focuser.
There are device hubs out on ASCOM Platform, which enable this behavior. Regarding the FW connected directly to your camera, connecting the camera directly through its ASCOM driver, you should be able to use the FW as well (as you could use the Focuser). But not at the same time NINA connects it.
I currently have no clue to overcome this on ASCOM side. Perhaps somebody else did this.
You might use ALPACA Remote on your NUC to publish all devices as ALPACA could handle multiple clients at the same time. But it's a huge effort just having them displayed in MW4 as well.
Michel

**Reply by JDAstroPhoto** [reply]
Thank-you, yes I am using NINA.  Since the focuser and filter wheel are set and forget, I will just continue to set them in NINA.  Since I need Autofocus routine, it makes sense to use NINA anyway.
